sonar-csharp-plugin/src/test/java/org/sonar/plugins/csharp/CSharpSonarRulesDefinitionTest.java

```diff
@@ -87,7 +87,13 @@ public void test_security_hotspot_has_correct_type_and_security_standards() {
 
     RulesDefinition.Rule rule = repository.rule(SECURITY_HOTSPOT_RULE_KEY);
     assertThat(rule.type()).isEqualTo(RuleType.SECURITY_HOTSPOT);
-    assertThat(rule.securityStandards()).containsExactlyInAnyOrder("cwe:502", "owaspTop10:a8", "owaspTop10-2021:a8");
+    assertThat(rule.securityStandards()).containsExactlyInAnyOrder(
+      "cwe:502",
```

Comment:
I had to update this since the standards are supported in `sonar-plugin-api` `9.9`